src/components/Tooltip/index.tsx

```diff
@@ -15,14 +21,20 @@ interface TooltipProps extends Omit<PopoverProps, 'content'> {
 
 interface TooltipContentProps extends Omit<PopoverProps, 'content'> {
   content: ReactNode
+  Container?: ({ children }: { children: ReactNode }) => JSX.Element
 }
 
 export default function Tooltip({ text, ...rest }: TooltipProps) {
   return <Popover content={<TooltipContainer>{text}</TooltipContainer>} {...rest} />
 }
 
-function TooltipContent({ content, ...rest }: TooltipContentProps) {
-  return <Popover content={<TooltipContainer>{content}</TooltipContainer>} {...rest} />
+function TooltipContent({ content, Container, ...rest }: TooltipContentProps) {
+  return (
+    <Popover
+      content={Container ? Container({ children: content }) : <TooltipContainer>{content}</TooltipContainer>}
```

Comment:
you _can_ use jsx here and default the argument to `TooltipContainer` to avoid the ternary

Reply:
i like using jsx, the default arg definitely cleansi tup

Reply:
I did update to a `wrap` boolean